plugins/security/nexus-ldap-plugin/src/test/java/org/sonatype/nexus/ldap/internal/LdapTestSupport.java

```
@@ -172,7 +172,13 @@ protected synchronized void resumeLdapServer(final String name) throws Exception
 
   @After
   public void stopLdap() throws Exception {
-    lookup(CacheManager.class).shutdown();
+    try {
+      lookup(CacheManager.class).close();
+    }
+    catch (Exception e) {
+      util.getLog().warn("Failed to shutdown cache-manager", e);
```

why mask this exception please? Could potentially hide real problems...